gradle.properties

```diff
@@ -8,7 +8,7 @@ toolkitVersion=2.11-SNAPSHOT
 publishToken=
 publishChannel=
 
-ideProfileName=2023.3
+ideProfileName=2024.1
```

Comment:
do we want to keep this 2023.3 until we move to 241 stable?

Reply:
sure